core/meta/src/TClass.cxx

```diff
@@ -5464,29 +5464,24 @@ TClass *TClass::LoadClassCustom(const char *requestedname, Bool_t silent)
 
 void TClass::LoadClassInfo() const
 {
-   R__LOCKGUARD(gInterpreterMutex);
+   // Load class info if not already loaded and auto-parsing is not suspended
+   if (!fClassInfo && !gInterpreter->IsAutoParsingSuspended()) {
```

Comment:
> What do I need to avoid when auto parsing is suspended? 
> if (!fClassInfo && !gInterpreter->IsAutoParsingSuspended()) {

The point I am trying to make is that if fClassInfo is a nullptr and fCanLoadClassInfo is true and
IsAutoParsingSuspend is true, then there is still case where fClassInfo can be set (because the AST already contains the information.

>  What do I need to avoid when auto parsing is suspended? 

Just the setting of fCanLoadClassInfo and the warning about the information missing since in this case the lookup failure (if any) was not fully authoritative ...